File: Programming Basics with Python/Вложени цикли/building.py

level = int(input())
room= int(input())

flat_num = 0
flat = ""
for levels in range(level, 0 ,-1):
    for rooms in range(0, room):
        flat_num = levels * 10 + rooms

        if levels == level:
            flat = f"L{flat_num}"
        elif level % 2 == 0:
            flat = f"A{flat_num}"
        elif level % 2 != 0:
            flat = f"O{flat_num}"
        
        print(flat, end=" ")

    print()

File: Programming Basics with Java/Nested Loops/Building.java
import java.util.Scanner;

public class Building 
{
    public static void main(String[] args) 
    {
        Scanner input = new Scanner(System.in);

        int levels = Integer.parseInt(input.nextLine());
        int rooms = Integer.parseInt(input.nextLine());

        for (int level = levels; level >= 1 && level <= levels; level--)
        {
            for (int room = 0; room < rooms; room++)
            {
                if (level == levels)
                {
                    System.out.print("L" + level + room + " ");
                }
                else if (level % 2 != 0)
                {
                    System.out.print("A" + level + room + " ");
                }
                else if (level % 2 == 0)
                {
                    System.out.print("O" + level + room + " ");
                }
            }

            System.out.println();
        }
    }
}
